pytroll_collectors/scisys.py

```diff
@@ -503,9 +503,19 @@ def stop(self):
         self.loop = False
 
 
+def update_sensor_tuple_or_list(to_send, config):
+    """Update sensor tuples and lists to MULTIPLE_SENSORS."""
+    to_send_dict = to_send.copy()
+    if "sensor" in to_send:
+        if type(to_send["sensor"]) is list or type(to_send["sensor"]) is tuple:
```

Comment:
I think it is perhaps more pythonic to do `isinstance(to_send["sensor"], collections.abc.Sequence)`
?

Reply:
`isinstance(to_send["sensor"], (list, tuple))` should also work